src/main/java/com/danielagapov/spawn/Services/S3/S3Service.java

```diff
@@ -47,7 +47,7 @@ public String putObject(byte[] file, String key) {
 
     public String putObject(byte[] file) {
         String key = UUID.randomUUID().toString();
-        return putObject(file, key);
+        return putObjectWithKey(file, key);
     }
 
     public UserDTO putObjectWithUser(byte[] file, UserDTO user) {
```

Comment:
```suggestion
    public UserDTO putProfilePictureWithUser(byte[] file, UserDTO user) {
```
The other methods you wrote are very extensible to other types of objects than only profile pictures, but this one is highly coupled with the idea of a user's profile picture, so I think it makes more sense to name it with that in mind, rather than a name that implies it is more generic for any type of object.